maskrcnn_benchmark/layers/sigmoid_focal_loss.py

```diff
@@ -0,0 +1,70 @@
+import torch
+from torch import nn
+# from torch.autograd import Function
+# from torch.autograd.function import once_differentiable
+
+# from maskrcnn_benchmark import _C
+
+"""
+class _SigmoidFocalLoss(Function):
```

Comment:
question: did your last experiments used the CUDA implementation or the Python implementation?
Could we maybe remove the CUDA code if you used the Python one? I'm thinking about using `torch.jit.script` to fuse everything into a single kernel, so we shouldn't see much performance penalty.